modules/drools/pom.xml

```diff
@@ -27,5 +27,6 @@
     <modules>
         <module>drools-checker</module>
         <module>rule-engine-testing</module>
+        <module>rules-test-development-project</module>
```

Comment:
this also has to be blacklisted in https://github.com/eclipse/antenna/blob/b0a52865794f96a0af8f5d63165dcad4dea65ff6/Jenkinsfile.eclipse#L129-L133

Reply:
This line looks weird, because all the antenna-testing paths are not existing anymore?

Reply:
oh yes, after https://github.com/eclipse/antenna/pull/243 has been merged it has to be blacklisted there.